src/views/document/RouteView.vue

```diff
@@ -192,11 +218,10 @@ export default {
 .automatic-gears {
   margin-bottom: 1.5rem;
 }
-
 @media print {
   .protection-area-info {
     margin: 0rem !important;
     padding: 0rem !important;
   }
 }
-</style>
+</style>
```

Comment:
please leave empty line at end of file

Reply:
OK, is it always mandatory to do so ?

Reply:
It's not mandatory, and in general has no consequences except when used with some old tools, but a good practice.